src/pypowsybl_jupyter/networkexplorer.py

```diff
@@ -126,32 +138,60 @@ def on_text_changed(d):
         value=sel_ctx.get_selected(),
         description='Found',
         disabled=False,
-        layout=widgets.Layout(width='350px', height='670px')
+        layout=widgets.Layout(flex='80%', height='100%', width='350px', margin='0 0 0 0')
     )
 
     def on_selected(d):
         if d['new'] != None:
             sel_ctx.set_selected(d['new'])
-            update_nad_diagram()
-            update_sld_diagram()
+            update_explorer()
 
     found.observe(on_selected, names='value')
 
-    update_nad_diagram()
-    update_sld_diagram()
+    history = widgets.Select(
```

Comment:
once you merge the latest changes from main could you update the docs screenshots to have that history widget in them?

Reply:
merged from main and updated the screenshots